client/src/app/app.module.ts

```diff
@@ -40,7 +42,8 @@ import { CommonModule, NgFor, NgIf, UpperCasePipe } from '@angular/common';
       { path: 'counter', component: CounterComponent },
       { path: 'fridge', component: FridgeComponent },
       { path: 'fetch-data', component: FetchDataComponent },
-      { path: 'recipe', component: RecipeComponent },
+      { path: 'recipe/recipe-list/:searchText', component: RecipeListComponent},
```

Comment:
There shouldn't be a separate path for this component. The list should be populated within the search component. A separate component is good but it should live under the search component.